akka-remote/src/main/scala/akka/remote/artery/LogByteStringTools.scala

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
I copied this from akka-http for debugging purposes. I guess we could put it somewhere in the internal stream (testing?) utils?

Reply:
Or just remove it again.

Reply:
it's a cool utility, I removed it from here but you should add it to akka streams somewhere